src/utils/request.ts

```diff
@@ -51,6 +56,9 @@ const errorHandler = (error: { response: Response }): Response => {
 const request = extend({
   errorHandler, // 默认错误处理
   credentials: 'include', // 默认请求是否带上cookie
+  headers: {
+    'X-API-KEY': fetchAPIKey()
```

Comment:
emm `IConfig` is just a typing, we could use:

```js
import {API_KEY} from '../../config/config'

headers: {
    'X-API-KEY': API_KEY
}
```

directly here.

Reply:
and because the APISIX doesn't support fetching APIKEY , so we could use the fixed value now.

Reply:
sorry, there should be

![image](https://user-images.githubusercontent.com/2106987/76921399-f3d99800-6908-11ea-84b2-f47edbb6dc12.png)

We just exported the API_KEY standalone, do not need put it in the default config value.

Reply:
Thank you for the detailed explanation!